rssparser/src/androidMain/kotlin/com/prof18/rssparser/RssParserBuilder.kt

```diff
@@ -0,0 +1,38 @@
+package com.prof18.rssparser
+
+import com.prof18.rssparser.internal.AndroidXmlParser
+import com.prof18.rssparser.internal.JvmXmlFetcher
+import kotlinx.coroutines.Dispatchers
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import java.nio.charset.Charset
+
+
+/**
+ * A Builder that creates a new instance of [RssParser]
+ *
+ * @property callFactory A custom [OkHttpClient] that can be provided by outside.
```

Comment:
```suggestion
 * @property callFactory A custom [OkHttpClient] that can be provided by the caller.
```

Is `callFactory` a good name if you expect a custom `OkHttpClient` here? 🤔